### PR TITLE
DAOS-11278 docs: add "Tools for debugging network connectivity issue"

### PR DESCRIPTION
**Pull request by wiliamhuang**
Provide a list of tools for debugging network connectivity issue
Provide a list of tools for measuring network latency and bandwidth

Required-githooks: true

Signed-off-by: Lei Huang <lei.huang@intel.com>

### CODE REVIEW

**Comment by wangshilong** [comment]
A small tip for doc update: it is a good idea to include "Doc-only: true" in the commit to skip unnecessary CI testing.

**Comment by wiliamhuang** [comment]
> A small tip for doc update: it is a good idea to include "Doc-only: true" in the commit to skip unnecessary CI testing.

Thanks you! Right. I forgot it in the first commit and added it immediately in the following commit. :)

**Comment by wiliamhuang** [comment]
@frostedcmos @Michael-Hennecke Could you please review? Thank you very much!

**Review by wangshilong — DISMISSED** [review]
Looks useful for me, i guess one good thing to have could be some common tools to benchmark network latency and  bandwidth except lnet_selftest.

**Comment by wangshilong** [comment]
> @frostedcmos @Michael-Hennecke Could you please review? Thank you very much!

Alex is still on Vacation until Oct...Michael might help review.

**Comment by wiliamhuang** [comment]
> Looks useful for me, i guess one good thing to have could be some common tools to benchmark network latency and bandwidth except lnet_selftest.

Thanks! ok. I can add example commands for some other related tools.

**Review by wangshilong — DISMISSED** [review]

**Comment by wiliamhuang** [comment]
@Michael-Hennecke Could you please review it? Thank you!

File: docs/admin/troubleshooting.md [hunk 1]
```diff
@@ -959,6 +959,136 @@ To configure a Syslog daemon to resolve the delivery errors and receive messages
...
+```
+client_node $ ping -c 3 -I eth1 10.165.192.121
+PING 10.165.192.121 (10.165.192.121) from 10.165.192.2 ens102: 56(84) bytes of data.
+64 bytes from 10.165.192.121: icmp_seq=1 ttl=64 time=0.177 ms
+64 bytes from 10.165.192.121: icmp_seq=2 ttl=64 time=0.120 ms
+64 bytes from 10.165.192.121: icmp_seq=3 ttl=64 time=0.083 ms
+```
+Make sure ping can reach the NIC your DAOS server is bound to.
+
+### fi_pinpong
```
Comment (Michael-Hennecke) [inline docs/admin/troubleshooting.md:1024]: pinpong -> pingpong

**Review by Michael-Hennecke — COMMENTED** [review]
What we are still missing is a tool that goes beyond simple point-to-point tests... All the tools mentioned here only check a single p2p connection. That's generally useful, but too cumbersome to systematically test a larger installation...

**Comment by wiliamhuang** [comment]
> What we are still missing is a tool that goes beyond simple point-to-point tests... All the tools mentioned here only check a single p2p connection. That's generally useful, but too cumbersome to systematically test a larger installation...

Thank you! I will add some info about IMB-P2P and intel cluster checker then.

File: docs/admin/troubleshooting.md (hunk 1 above)
Comment (wiliamhuang) [reply docs/admin/troubleshooting.md]: Fixed. Thank you!

**Comment by wiliamhuang** [comment]
Added info for the tools to diagnose network issues for a large cluster.

**Review by wangshilong — APPROVED** [review]

**Comment by wiliamhuang** [comment]
@Michael-Hennecke Could you please review this PR? Thank you!

**Comment by wiliamhuang** [comment]
@Michael-Hennecke @mchaarawi Could you please review this PR for docs? Thank you!

**Review by soumagne — APPROVED** [review]

**Comment by ipoddubn** [comment]
@daos-stack/daos-gatekeeper please expedite the landing of this PR. thank you!

**Pull request merged**